src/widgets/stats/stats.tsx

```diff
@@ -74,19 +78,50 @@ export type StatsWidget = WidgetFactory<
 >;
 
 export const defaultTemplates: StatsComponentTemplates = {
-  text: `
-    {{#areHitsSorted}}
-      {{#hasNoSortedResults}}No relevant results{{/hasNoSortedResults}}
-      {{#hasOneSortedResults}}1 relevant result{{/hasOneSortedResults}}
-      {{#hasManySortedResults}}{{#helpers.formatNumber}}{{nbSortedHits}}{{/helpers.formatNumber}} relevant results{{/hasManySortedResults}}
-      sorted out of {{#helpers.formatNumber}}{{nbHits}}{{/helpers.formatNumber}}
-    {{/areHitsSorted}}
-    {{^areHitsSorted}}
-      {{#hasNoResults}}No results{{/hasNoResults}}
-      {{#hasOneResult}}1 result{{/hasOneResult}}
-      {{#hasManyResults}}{{#helpers.formatNumber}}{{nbHits}}{{/helpers.formatNumber}} results{{/hasManyResults}}
-    {{/areHitsSorted}}
-    found in {{processingTimeMS}}ms`,
+  text({
+    areHitsSorted,
+    hasNoSortedResults,
+    hasOneSortedResults,
+    hasManySortedResults,
+    hasNoResults,
+    hasOneResult,
+    hasManyResults,
+    nbSortedHits,
+    nbHits,
+    processingTimeMS,
+  }) {
+    let sentence = '';
```

Comment:
I'd like this more if it's extracted in a function so that instead of reassigning the variable, it can be different return statements, what do you think?

Reply:
Does something like [this](https://github.com/algolia/instantsearch.js/pull/5092/commits/e2389c1c46f0bf00756886b581da8dbdeee77ddd) works better for you?